Ktisis/Interface/Components/BoneTree.cs

```diff
@@ -32,30 +49,49 @@ public class BoneTree {
 				}
 			}
 		}
+
+		private static bool BoneDoesMatch(Bone bone) => 
+			SearchText != "" && (new [] { bone.UniqueName, bone.LocaleName }).Any(
+				name => name.Contains(SearchText, StringComparison.CurrentCultureIgnoreCase)
+			);
+
 		private static void DrawBoneTreeNode(Bone bone) {
 			if (bone == null) return;
 
 			var children = bone.GetChildren();
+			var decendents = bone.GetDescendants();
+
+			bool hasChildInQuery = decendents.Any(BoneDoesMatch);
```

Comment:
I would check for `SearchText != ""` outside of the any call here, like this:

```suggestion
			bool hasChildInQuery = SearchText != "" && decendents.Any(BoneDoesMatch);
```

Doing so prevents an unnecessary iteration over the entire array if there's no search text entered.